src/dndbeyond/content-scripts/character.js

```diff
@@ -204,7 +204,7 @@ function rollHitDie(multiclass, index) {
     });
 }
 
-function rollItem(force_display = false, force_to_hit_only = false, force_damages_only = false, spell_group = null) {
+async function rollItem(force_display = false, force_to_hit_only = false, force_damages_only = false) {
```

Comment:
your fix to the merge conflict here is wrong, removing the `spell_group` argument.
Also, not sure why changing the function to async is needed anymore